week-4/F-object-keys/exercise-part-1.js

```diff
@@ -15,10 +16,13 @@ var mentorsAges = {
 
 // ONLY EDIT BELOW THIS LINE
 
-var mentorsNames = ;
+var mentorsNames = Object.keys(mentorsAges) ;
 
-var mentorsNamedCapitalised = ;
+var mentorsNamedCapitalised = mentorsNames.map(getName); ;
 
+function getName(mentorsName) {
+  return mentorsName.toUpperCase();
+}
```

Comment:
`getName` isn't the best name for this function, because it does something more than simply getting the name.